azure-pipelines.yml

```diff
@@ -175,31 +175,32 @@ stages:
             # the unusual install method. For the bug report about missing .pc
             # files: https://github.com/MacPython/openblas-libs/issues/74
             set -euo pipefail
-            docker pull i386/ubuntu:bionic
-            docker run -v $(pwd):/scipy i386/ubuntu:bionic /usr/bin/linux32 /bin/bash -c "cd scipy && \
+            docker pull i386/ubuntu:focal
+            docker run -v $(pwd):/scipy i386/ubuntu:focal /usr/bin/linux32 /bin/bash -c "cd scipy && \
+            export DEBIAN_FRONTEND=nonintercative && \
             apt-get -y update && \
-            apt-get -y install curl python3.8-dev python3.8 python3-distutils pkg-config libpng-dev libjpeg8-dev libfreetype6-dev && \
+            apt-get -y install curl python3.9-dev python3.9 python3-distutils pkg-config libpng-dev libjpeg8-dev libfreetype6-dev && \
             curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
-            python3.8 get-pip.py && \
+            python3.9 get-pip.py && \
             pip3 --version && \
-            pip3 install setuptools==59.6.0 wheel numpy==1.19.5 cython==0.29.21 pybind11 pytest pytest-timeout pytest-xdist pytest-env pytest-cov Pillow mpmath pythran pooch && \
+            pip3 install setuptools==59.6.0 wheel numpy==1.21.6 cython==0.29.21 pybind11 pytest pytest-timeout pytest-xdist pytest-env pytest-cov Pillow mpmath pythran pooch && \
             apt-get -y install gcc-8 g++-8 gfortran-8 wget && \
```

Comment:
I think it'd be much easier to have:

```
apt-get -y install build-essential gfortran wget && \
```

and remove `CC=gcc-8 CXX=g++-8 F77=gfortran-8 F90=gfortran-8` on the `setup.py install` line.